components/RightSidebar.js

```diff
@@ -1,30 +1,59 @@
-import styles from "../styles/Home.module.css";
+import styles from "../styles/Sidebar.module.css";
+import Link from "next/link";
 
-export default function RightSidebar() {
+export default function RightSidebar({ screenings, movies, movie }) {
```

Comment:
Having two separate properties for plural (`movies`) and singular (`movie`) content poses some challenges. For example, what happens if the user of the component provides both?

There are better ways of achieving this, e.g. creating two components, one for the singular case and one for the plural, which internally use the same general-purpose component where most of the shared logic and UI resides.